docs/rst/manual/getting_started/configuration/poly_new/guide.rst

```diff
@@ -0,0 +1,291 @@
+
+Run an empty experiment
+***********************
+To run an empty experiment, select "Ensamble experiment" as the Simulation mode. Then press the "Start simulation" button. You get a dialog asking if you want to start with the "default" case. That is fine, so press "Yes". The experiment will run, but since we have configured no forward model, only an empty experiment is executed.
+
+A new window will appear called "Simulations". Here you can see the status of the running of the experiment. If you press "Details", you can see status of each running realization.
+
+.. image:: minimal/simulations.png
+
+As it's running you can see that not all the realizations are running at the
+same time. There is an upper limit to how many realizations can run at the same
+time, which is decided by the queue. You will probably have 2 running at a time.
+We will configure this at a later stage. When all the realizations are done,
+close all the ERT windows.
+
+Inspect the results
+*******************
+Now in the project folder there should be two new folders, "simulations" and "storage". Storage is ERT's internal data, and should not be touched. The "simulations" folder is created based on the RUNPATH configuration. If you open this folder you will see one folder for each realization, labeled realization0, realization1, etc. Inside each of these folders is where each realization has run. If you look inside, you will see a few files that ERT uses to keep track of the jobs. 
+
+* jobs.json: defines the jobs to run
+* OK: indicates success. On error EXIT and ERROR file is created instead.
+* STATUS: legacy status file
+* status.json: used to communicate status to the ERT user interface.
+
+You should not touch these files, either manually, or as part of the jobs your experiments define.
+
+Adding a Forward Model and Runpath
+----------------------------------
+The example project will use a simple polynomial evaluation as the forward model. In real experiments this would usually be running a simulator, like Eclipe, instead. The forward model is implemented as a simple python script "poly_eval.py". We will start with a simple script just to check that we can run it. Later we will expand on it and the configuration file to introduce some fundamental features of ERT.
+
+Add a Runpath
+*************
+In the minimal config, we used the default :ref:`RUNPATH <runpath>` which ran the realization in simulations/realization%d. This is not sufficient for running the algorithms of ERT. When running the algorithms of ERT, we must usually run the Ensamble several times, and the runpath needs to reflect this. We need to have each iteration run in a different forder, so that they won't step on each others toes. Add to the config ("poly.ert") file the following line::
+
+    RUNPATH poly_out/real_%d/iter_%d
+
+The second "%d" in the runpath will be replaced by the iteration of running the ensamble. In this way we can run several times, and have the results in different folders.
```

Comment:
Is this true? Might be me misunderstanding what you're trying to convey. Running several times is handled by the case name, the iterations is for the iterative algorithms.

Reply:
For some definitions of all the words, it is true. Alternatively for some definition of true.